---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER | Microsoft Docs
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
author: stevestein
ms.author: sstein
ms.openlocfilehash: d0354ba6d980b359778a66fc13b65fe233cacba1
ms.sourcegitcommit: e042272a38fb646df05152c676e5cbeae3f9cd13
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 04/27/2020
ms.locfileid: "67996002"
---
# <a name="localdb_error_cannot_access_instance_folder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Detalhes  
  
|||  
|-|-|  
|Nome do Produto|SQL Server|  
|ID do evento|261|  
|Origem do Evento|Runtime de banco de dados local do SQL Server 12.0|  
|Componente|API do runtime de banco de dados local|  
|Texto da mensagem|Não é possível acessar a pasta da instância do banco de dados local:%% LOCALAPPDATA\\ %% \\>Microsoft\Microsoft SQL Server DB\Instances local<nome da instância.|  
  
## <a name="explanation"></a>Explicação  
 Não é possível acessar uma pasta de instância.  
  
## <a name="user-action"></a>Ação do usuário  
  
